collectors/python.d.plugin/beanstalk/metadata.yaml

```diff
@@ -0,0 +1,207 @@
+name: beanstalk
+title: TBD Monitoring
+duplicate_for_virtual_integrations:
+  list:
+    - '[TBD](link to TBD)'
+    - '[TBD](link to TBD)'
+overview:
+  how_the_collector_works:
+        # The CI will remove the curly brackets in the link, they are for replacing with the virtual integrations, to replace the link each time
+    description: |
+      This collector works by... {[TBD](link to TBD)} ...
+  supported_platforms:
+    description: |
+      If this collector can't run on some platforms specify them here. Leave empty otherwise.
+  multi-instance: true
+  permissions:
+    description: |
+      if there are any notable permission requirements, eg running as root and so on, specify them here. Leave empty otherwise.
+setup:
+  prerequisites:
+    list:
+      - title: Prerequisite title for heading
+        text: |
+          - TBD
+          - TBD
+  behavior:
+    auto_detection:
+      description: |
+        TBD
+    limits:
+      description: |
+        TBD
+    performance_impact:
+      description: |
+        TBD
+  configuration:
+    file:
+      description: |
+        which file the user has to edit. Example: go.d/apache.conf
+    options:
+      description: |
+        TBD, use | and newline for multiline
+      folding:
+        title: Config options
+        enabled: true
+      list:
+        - name: option name
+          description: option description
+          default: a number or a string
+          required: true or false
+    examples:
+      folding:
+        title: Config
+        enabled: false
+      list:
+        - name: Basic / else describe what this example is about
+          description: |
+            A basic example configuration / describe your example
+          data:
+            jobs:
+              - name: example
+                TBD: TBD
+troubleshooting:
+  problems:
+    list:
+      - name: possible troubleshooting title
+        text: |
+          describe the issue and the troubleshooting process
+related_resources:
+  integrations:
+    list: [apache, weblog, httpcheck, etc]
+        # This text will be added in the overview section
+    overview_text: |
+      give a simple description if there are any other collectors related to this collector. Example -> To get the most out of TBD, you can do XYZ with collectors X Y and Z. Check our related resources for more information.
+    provide:
+      description: |
+        What info I can provide for other plugins that reference me? this will only be visible in their end, they will scrape this text. Example -> XYZ allows monitoring X Y Z metrics...
+alerts:
+  - name: beanstalk_server_buried_jobs
+    link: https://gihtub.com/netdata/netdata/blob/master/health/health.d/beanstalkd.conf
+    metric: beanstalk.current_jobs
```

Comment:
i dont think `beanstalk.current_jobs` is a metric - i think its a chart. This alarm seems to be operating on the `buried` dimension of the `beanstalk.current_jobs` chart.

so am wondering what we mean by `metric: beanstalk.current_jobs` ?

Reply:
maybe we mean that the metric is something like `beanstalk.current_jobs.buried` but not sure we really have any such standard representation like this strictly speaking.

Reply:
That comes from https://github.com/netdata/netdata/blob/master/health/health.d/beanstalkd.conf#L4, it is autogenerated, by parsing the .conf files

Reply:
Keep in mind this is the template, we can always not render stuff at the final presentation, for example the `os` field in alerts is just for manually curating some system overview modules that end up having metrics but not the alerts in different OSes. That field will not make the end presentation, it will be there so we can remove any wrong alerts.

Same can happen with `metric` field, it is getting whatever the .conf files say